proposals/lip-0038.md

```diff
@@ -84,7 +84,7 @@ Each aggregate signature needs to be accompanied by some information that specif
 
 ```python
 def createAggSig(keysList: list[bytes], pubKeySignaturePairs: list[tuple[bytes, bytes]]) -> tuple[bytes, bytes]:
-    aggregationBits = byte string of length ceil(length(keyList)/8) with all bytes set to 0
+    aggregationBits = byte string of length ((length(keysList)+8-1) //8) with all bytes set to 0
```

Comment:
```suggestion
    aggregationBits = byte string of length (length(keysList) + 7)  // 8 with all bytes set to 0
```

Reply:
Also consider adding a dedicated function for `ceiling` similar to LIP 57 https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#ceiling.